lib/services/obm-api-service.js

```diff
@@ -0,0 +1,229 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+
+module.exports = obmApiServiceFactory;
+di.annotate(obmApiServiceFactory, new di.Provide('Http.Services.Api.Obms'));
+di.annotate(obmApiServiceFactory,
+    new di.Inject(
+        'Promise',
+        '_'
+    )
+);
+function obmApiServiceFactory(
+    Promise,
+    _
+
+) {
+
+    function ObmApiService() {
+    }
+
+    var obms = [
```

Comment:
@benbp Yep.  This is some of the technical debt that we discussed last night.  We've created some stories in our backlog to go back and clean this stuff up.